mne_microstates.py

```diff
@@ -67,6 +68,12 @@ def segment(data, n_states=4, n_inits=10, max_iter=1000, thresh=1e-6,
         function is called.
     verbose : int | bool | None
         Controls the verbosity.
+    return_best_gev : bool
+        Option to return best golbally explained variance. Defaults to False.
+    weight_array : array-like | None
+        This is an optional array which can be used to weight during the adapted
+        k-means algorithm. It should have the same length as the number of samples
+        in the data. Defaults to None.
```

Comment:
In my previous comment, I suggested moving the `return_best_gev` parameter, in which case it must be moved in the docstring as well.